members/state_ma_gov_charlie_baker.yaml

```diff
@@ -60,6 +62,8 @@ contact_form:
     - click_on:
         - value: Submit
           selector: input#fsSubmitButton1744869
+    - find:
```

Comment:
What's this final find doing?

Reply:
looks for the element that contains the success msg. I want to make sure yamltron sees text